client/src/app/shared/ProductItem/ExtendedDetails.js

```diff
@@ -117,8 +120,13 @@ class ViewDetailsFull extends Component {
             />
             <h4 className="highlighted">{title}</h4>
           </Link>
-          <div className={styles.full__sizes}>{`sizes : ${allSizes}`}</div>
-          <div className={styles.full__swatches}>{swatches}</div>
+          <div className={styles.full__sizes}>
+            <div className={styles.sizesText}>Sizes: </div>
```

Comment:
So many `<div>`, why not use more semantic elements (`<p>`, `<h3>`)?

Reply:
ok, i'll change

Reply:
done